eos/constraints/B-to-D-pi-l-nu.yaml

```diff
@@ -692,6 +692,136 @@ B^0->(Dpi)^+l^-nubar::KinematicDistributionsAngularObservables@BBGJvD:2021A:
      - [-3.4434350614771665e-07, 9.247471362904429e-06, -4.975955661374302e-06, -2.5089096178472918e-05, -4.117674206856538e-06, -2.622718889971824e-06, 1.0103029255665281e-05, -1.1665881221778744e-05, 9.961498357338163e-07, 1.0112330419560555e-05, 4.807834726665146e-08, -6.845366978535433e-07, 1.7898421496094407e-06, 3.1935446948673432e-06, -1.285688955790833e-07, 4.754795249225177e-06, -1.1533133836023689e-05, -3.17815451725826e-05, -8.816323526892334e-06, -3.947855542185224e-06, 1.1791996158797502e-05, -7.435519058147198e-06, 5.221607575357882e-06, 2.068372187955312e-05, 1.1088749022291992e-06, -2.330222521524111e-06, 5.347921678514e-06, 2.1261412084259977e-05 ]
 # }}}
 
+############
+# Belle II #
+############
+# {{{
+B^0->D^*+l^-nu::KinematicDistribution[w]@Belle-II:2023C:
+  type: MultivariateGaussian(Covariance)
+  observables:
+    - B->D^*lnu::P(w_min,w_max)
```

Comment:
I would like to generalize the use of lists to declare the maps in yaml:
```yaml
observables:
  [ B->D^*lnu::P(w_min,w_max), B->D^*lnu::P(w_min,w_max),... ]
```
This will allow us to keep relatively dense and concise files.

Reply:
Fixed.

Reply:
It is not fixed, is it?

Reply:
I will try to check later today. Wonder how I lost it.

Reply:
Had not saved the changes on the computer at home. Fixed now!